hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1332,6 +1332,9 @@ public static boolean omInit(OzoneConfiguration conf) throws IOException,
       AuthenticationException {
     OMHANodeDetails omhaNodeDetails = OMHANodeDetails.loadOMHAConfig(conf);
     String nodeId = omhaNodeDetails.getLocalNodeDetails().getNodeId();
+    // Checking certificate duration validity by using
+    // validateCertificateValidityConfig() in SecurityConfig constructor.
+    new SecurityConfig(conf);
```

Comment:
```new SecurityConfig(conf);``` is doing much more than just validating, can we refactor the certificate code extraction part & combine with the validation method into a single method & call that here?

Reply:
In the SecurityConfig constructor we're mostly initializing the member variables and most of them are used in `validateCertificateValidityConfig` so I don't think there would be too much extra load if we call out an object. If we try to extract a method for initializing variables related to `validateCertificateValidityConfig` then also we have to explicitly initialize each property being used in `validateCertificateValidityConfig`, what do you suggest?